CHANGELOG.md

```diff
@@ -64,6 +37,34 @@ preflightTest.on('completed', (report: PreflightTestReport) => {
   console.log(" Your estimated quality score was " + report.qualityScore);
 });
 ```
+
+The [PreflightTestReport](https://sdk.twilio.com/js/video/releases/2.17.0/docs/global.html#PreflightTestReport) generated by `preflightTest` on `completed` provides statistics that can be used to determine the quality of the connection. Some of the useful stats and their optimal values are listed below.
+
+| Stat                     |  Description | Optimal values
+| -------------------------|--------------|------------------|
+| jitter                   | Packets delay variation on audio tracks | < 30ms
+| packetLoss               | Packet loss as a percent of total packets sent | < 1%
+| rtt                      | Round trip time in milliseconds. | < 200ms
+
```

Comment:
Should there be a bitrate stat too?

Reply:
we only use jitter/packetLoss and rtt to compute quality score - and they also have an "optimal value" associated with them. Although the generated report has bitrates - they are not used for any scoring, as we are not using full available bandwidth.

Reply:
ok, fair enough